src/cli.test.js

```diff
@@ -123,7 +130,7 @@ test("adds explicit `--save-dev` flag when using `-D, -d, --dev` with NPM", asyn
       flags.map(flag => getCliInstallCommand(["eslint-config-airbnb", flag]))
     );
     commands.forEach((command, i) =>
-      t.equal(/ --save-dev /.test(command), true, `flag: \`${flags[i]}\``)
+      t.equal(/ --save-dev\b/.test(command), true, `flag: \`${flags[i]}\``)
```

Comment:
what's the `\b` fixing?

Reply:
I was getting intermittent test failures because sometimes —save-dev was on the end of a line and so didn’t have a trailing space. The other tests were configured this way.